contracts/src/TeleporterTokenDestination.sol

```diff
@@ -34,8 +34,8 @@ abstract contract TeleporterTokenDestination is
     /// @notice The ERC20 token this contract uses to pay for Teleporter fees.
     address public immutable feeTokenAddress;
 
-    /// @notice Required gas limit for sending tokens to another chain.
-    uint256 public constant SEND_TOKENS_REQUIRED_GAS = 220_000;
+    /// @notice Required gas limit for sending tokens back to the source blockchain.
```

Comment:
A few asks for this comment:
- Can you update this comment to describe the multihop case? Right now, it reads more like a point to point transfer.
- Can you clarify that this figure accounts for the fixed gas relaying the transfer to a third chain?  Also can you mention what blockchain this gas is spent on?

Reply:
updated comment to clarify